charts/simple-app/README.md.gotmpl

```diff
@@ -12,6 +12,15 @@ defaults for you like the Kubernetes [Horizontal Pod Autoscaler][hpa].
 
 ## Upgrade Notes
 
+### 0.27.x -> 1.0.x
+
+**BREAKING: You need to explicitly set the `livenessProbe` and `readinessProbe`**
```

Comment:
Add required param text to values file.

Reply:
This looks duplicative of the other comment on values.yaml.

Reply:
⛔️ Blocking comment.

I feel that we should either add more text to the parameter in the values file explicitly stating that this is a required field and/or we should upgrade the required text output on failure to be a bit more human descriptive/helpful/breadcrumbs.

I feel strongly about this for OOBE reasons. Just because our current OOBE is not great does not mean that we should continue to allow not-so-great pattern matching. Lets strive to improve ourselves in _this_ PR. 🤷🏼 idk just my take. Didn't feel it was that hot as it should be quick docfix.

Reply:
Done.